proposals/templates/proposals/pdf/observation_v2.html

```diff
@@ -34,50 +34,50 @@
         <th>{% get_verbose_field_name "observations" "observation" "is_anonymous" %}</th><td>{{ observation.is_anonymous|yesno:_("ja,nee") }}</td>
     </tr>
     {% if observation.is_anonymous %}
-    <tr>
-        <th>{% get_verbose_field_name "observations" "observation" "is_anonymous_details" %}</th><td>{{ observation.is_anonymous_details }}</td>
-    </tr>
+        <tr>
+            <th>{% get_verbose_field_name "observations" "observation" "is_anonymous_details" %}</th><td>{{ observation.is_anonymous_details }}</td>
+        </tr>
     {% endif %}
     <tr>
         <th>{% get_verbose_field_name "observations" "observation" "is_in_target_group" %}</th><td>{{ observation.is_in_target_group|yesno:_("ja,nee") }}</td>
     </tr>
     {% if observation.is_in_target_group %}
-    <tr>
-        <th>{% get_verbose_field_name "observations" "observation" "is_in_target_group_details" %}</th><td>{{ observation.is_in_target_group_details }}</td>
-    </tr>
+        <tr>
+            <th>{% get_verbose_field_name "observations" "observation" "is_in_target_group_details" %}</th><td>{{ observation.is_in_target_group_details }}</td>
+        </tr>
     {% endif %}
     <tr>
         <th>{% get_verbose_field_name "observations" "observation" "is_nonpublic_space" %}</th><td>{{ observation.is_nonpublic_space|yesno:_("ja,nee") }}</td>
     </tr>
     {% if observation.is_nonpublic_space %}
-    <tr>
-        <th>{% get_verbose_field_name "observations" "observation" "is_nonpublic_space_details" %}</th><td>{{ observation.is_nonpublic_space_details }}</td>
-    </tr>
-    <tr>
-        <th>{% get_verbose_field_name "observations" "observation" "has_advanced_consent" %}</th><td>{{ observation.has_advanced_consent|yesno:_("ja,nee") }}</td>
-    </tr>
-        {% if not observation.has_advanced_consent %}
         <tr>
-            <th>{% get_verbose_field_name "observations" "observation" "has_advanced_consent_details" %}</th><td>{{ observation.has_advanced_consent_details }}</td>
+            <th>{% get_verbose_field_name "observations" "observation" "is_nonpublic_space_details" %}</th><td>{{ observation.is_nonpublic_space_details }}</td>
         </tr>
+        <tr>
+            <th>{% get_verbose_field_name "observations" "observation" "has_advanced_consent" %}</th><td>{{ observation.has_advanced_consent|yesno:_("ja,nee") }}</td>
+        </tr>
+        {% if not observation.has_advanced_consent %}
+            <tr>
+                <th>{% get_verbose_field_name "observations" "observation" "has_advanced_consent_details" %}</th><td>{{ observation.has_advanced_consent_details }}</td>
+            </tr>
```

Comment:
Hm, if I were an HTML formatter, I'd think this line is way too long, and would at the very least want to move the two elements to separate lines.

Ideally, I'd even change it to something like this:
```html
<tr>
    <th>
        x
    </th>
    <td>
        y
    </td>
</tr>
```

I guess djhtml is primarily an indenter :P
That's fine, I'm just used to Prettier also doing stuff like that for me.